src/openrct2/network/NetworkTypes.h

```diff
@@ -59,7 +59,7 @@ enum NETWORK_COMMAND
     NETWORK_COMMAND_PING,
     NETWORK_COMMAND_PINGLIST,
     NETWORK_COMMAND_SETDISCONNECTMSG,
-    NETWORK_COMMAND_GAMEINFO,
+    NETWORK_COMMAND_GAMEINFO = 9, // Must always be 9 for the master server.
```

Comment:
Wouldn't this make `NETWORK_COMMAND_GAMEINFO` = `NETWORK_COMMAND_SETDISCONNECTMSG` ?

Reply:
Right, we'll ill just move the new one down then.